lib/aptible/cli/subcommands/deploy.rb

```diff
@@ -29,6 +29,12 @@ def self.included(thor)
                    desc: 'Detach this app from its git repository: ' \
                          'its Procfile, Dockerfile, and .aptible.yml will be ' \
                          'ignored until you deploy again with git'
+            option :container_count, type: :numeric
+            option :container_size, type: :numeric
```

Comment:
Need descriptions on these two container options similar to `:container_profile`